composer.json

```diff
@@ -25,6 +25,7 @@
         "silverstripe/reports": "^4.7@dev",
         "silverstripe/siteconfig": "^4.7@dev",
         "silverstripe/versioned": "^1.7@dev",
+        "silverstripe/versioned-admin": "^1.7@dev",
```

Comment:
This is required to get CMS page history to work since we've migrated from the deprecated CMS class to the class in versioned-admin